ietf/doc/utils_search.py

```diff
@@ -143,14 +143,15 @@ def fill_in_document_table_attributes(docs, have_telechat_date=False):
     xed_by = RelatedDocument.objects.filter(target__name__in=list(rfc_aliases.values()),
                                             relationship__in=("obs", "updates")).select_related('target')
```

Comment:
```suggestion
    # Revisit this block after RFCs become first-class Document objects
    xed_by = list(RelatedDocument.objects.filter(
        target__name__in=list(rfc_aliases.values()),
        relationship__in=("obs", "updates")
    ).select_related('target'))
```

Reply:
All of this will become much easier to do (the database will be able to do it for us) after we make RFCs first-class Document objects and the database knows rfc numbers as integers.